src/gui/tray/UserModel.cpp

```diff
@@ -413,15 +413,23 @@ void User::slotAddError(const QString &folderAlias, const QString &message, Erro
     }
 }
 
+bool User::isValueableActivity(Folder *folder, const SyncFileItemPtr &item)
+{
+    // Check if we are adding it to the right account and if it is useful information (protocol errors)
+    const auto correctAccount = folder->accountState() == _account.data();
```

Comment:
@FlexW Any reason for not naming it **is**CorrectAccount ?

Reply:
no;)